### PR TITLE
Define get_users()

### PR DESCRIPTION
**Pull request by mirceaulinic**
Retrieves users configured on the device

### CODE REVIEW

File: napalm_base/base.py [hunk 1]
```diff
@@ -1009,3 +1009,30 @@ def get_probes_results(self):
...
         """
         raise NotImplementedError
+
+    def get_users(self):
+        """
+        Returns a dictionary with the configured users.
+        The keys of the main dictionary represents the username. The values represent the details of the user,
+        represented by the following keys:
+
+            * level (int)
```
Comment (dbarrosop) [inline napalm_base/base.py]: What if it's a juniper user? Should we just map the default classes to numbers? Something like:

operator:        1
read-only:       9
super-user:      15

Or should we do it the other way around, naming the levels somehow?

**Comment by mirceaulinic** [comment]
Indeed. Due to the fact that most network engineers are mostly used with the level rather than class name, I preferred to have this field and integer (between 0 and 15). On JunOS will define a map between the class name and level.

```python
_JUNOS_CLASS_CISCO_PRIVILEGE_LEVEL_MAP = {
  'super-user': 15,
  'superuser': 15,
  'operator': 5,
  'read-only': 1,
  'unauthorized': 0
}
```

**Comment by dbarrosop** [comment]
:shipit: 

Do you have any implementation already?

**Comment by mirceaulinic** [comment]
Yup :)

**Comment by mirceaulinic** [comment]
@dbarrosop Doing multiple things right now and I did a mess with the NTP peers, can we postpone it till tomorrow?

**Comment by dbarrosop** [comment]
Of course! : )

**Comment by mirceaulinic** [comment]
Meanwhile, would be possible to merge this one and have the necessary
environment prepared?

Le mar. 26 avr. 2016 18:26, David Barroso <notifications@github.com> a
écrit :

**Comment by dbarrosop** [comment]
Done!

**Comment by mirceaulinic** [comment]
Thanks! :)

>

**Comment by mirceaulinic** [comment]
@dbarrosop All setup & ready here.

**Pull request merged**